akka-docs/src/main/paradox/stream/stream-refs.md

```diff
@@ -0,0 +1,156 @@
+# (Remote) Akka Streams over Network boundaries 
+
+Stream references, or "stream refs" for short, allow running Akka Streams across multiple nodes within 
+an Akka Cluster. 
+
+Unlike heavier "streaming data processing" frameworks, Akka Streams do not require running and maintaining
+a cluster onto which the streams are later "deployed". Akka Stream Refs are, as the name implies, references
```

Comment:
Any preference how to call them in witten text? I swap between "Stream Refs" "stream references" etc..